lib/src/components/NxProgressBar/NxProgressBar.scss

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-progress-bar {
+  display: flex;
+  flex-direction: column;
+  gap: var(--nx-spacing-2x);
+  margin: var(--nx-spacing-6x) 0;
+}
+
+.nx-progress-bar--inline-counter {
+  flex-direction: row;
+}
+
+.nx-progress-bar--inline {
+  display: inline-flex;
+  margin: 0;
+}
+
+.nx-progress-bar__counter .nx-counter {
+  margin: 0;
+}
+
+.nx-progress-bar--success {
+  color: var(--nx-swatch-green-25);
+
+  .nx-counter {
+    border-color: var(--nx-swatch-green-25);
+    background-color: var(--nx-swatch-green-95);
+  }
+}
+
+.nx-progress-bar--error {
+  color: var(--nx-swatch-red-40);
+
+  .nx-counter {
+    border-color: var(--nx-swatch-red-40);
+    background-color: var(--nx-swatch-red-95);
+  }
+}
+
+.nx-progress-bar__label {
+  align-items: center;
+  display: flex;
+  gap: var(--nx-spacing-2x);
+  grid-area: label;
+
+  .nx-icon {
+    margin: 0 0 0 var(--nx-spacing-2x);
+  }
+}
+
+$normal-height: 25px;
+
+progress.nx-progress-bar__progress[value] {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+
+  border: 0;
+
+  height:$normal-height;
+  width: 100%;
+}
+
+// Track
+progress.nx-progress-bar__progress[value],
+progress.nx-progress-bar__progress[value]::-webkit-progress-bar {
+  background: var(--nx-swatch-grey-95);
+  border-radius: $normal-height;
+  height: $normal-height;
+  overflow: hidden;
+  width: 300px;
+}
+
+.nx-progress-bar--small {
+  progress.nx-progress-bar__progress[value],
+  progress.nx-progress-bar__progress[value]::-webkit-progress-bar {
+    width: 100px;
+  }
+}
+
+.nx-progress-bar--full {
+  progress.nx-progress-bar__progress[value],
+  progress.nx-progress-bar__progress[value]::-webkit-progress-bar {
+    min-width: 300px;
+    width: 100%;
+  }
+}
+
+.nx-progress-bar--inline {
+  progress.nx-progress-bar__progress[value],
+  progress.nx-progress-bar__progress[value]::-webkit-progress-bar {
+    height: 16px;
+    width: 150px;
+  }
+}
+
+// Indicator
+progress.nx-progress-bar__progress[value]::-moz-progress-bar {
+  background: var(--nx-swatch-teal-50);
+}
+
+progress.nx-progress-bar__progress[value]::-webkit-progress-value {
+  background: var(--nx-swatch-teal-50);
```

Comment:
Should be teal-60 according to sketch.  Also these two blocks can be combined to avoid repeating the color

Reply:
> Should be teal-60 according to sketch. Also these two blocks can be combined to avoid repeating the color

The color is fixed here: 12b27041fdcedd016145c83e35724bc0802e1aed

I had these two combined before, but for some reason that broke the styling and the only fix was having them separated like that. Going to investigate

Reply:
> I had these two combined before, but for some reason that broke the styling and the only fix was having them separated like that. Going to investigate

Very strange.  I'm seeing that too

Reply:
> Very strange. I'm seeing that too

Judging from [this](https://stackoverflow.com/questions/70550606/is-there-a-reason-why-css-pseudo-elementnon-standard-cannot-be-combined-with-c), I believe what's happening is any part of the selector includes a psuedo-element that Chrome doesn't recognize (like `::-moz-...`), Chrome will error out parsing the entire compound selector and not consider the half that it could have understood